examples/Tutorial.ipynb

```diff
@@ -0,0 +1,1521 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# An Introduction to SKLL"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "SKLL (Scikit-Learn Laboratory) provides a number of command-line utilities and an API that make it simpler to run scikit-learn machine learning experiments with pre-generated features.\n",
+    "SKLL can be used in 2 ways:\n",
+    "\n",
+    "\n",
+    "1. **Via the Command Line**\n",
+    "2. **Via the API**"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## 1. Using SKLL via the Command Line\n",
+    "\n",
+    "In this example, we will use the following four steps to understand how to use SKLL via the command line.\n",
+    "\n",
+    "- Get the data in a [SKLL compatible format](https://skll.readthedocs.io/en/latest/run_experiment.html#file-formats).\n",
+    "- Create a SKLL [experiment configuration file](https://skll.readthedocs.io/en/latest/run_experiment.html#create-config) for cross-validation.\n",
+    "- Run the machine learning experiment using the [*run_experiment*](https://skll.readthedocs.io/en/latest/run_experiment.html) command.\n",
```

Comment:
I italicize it when I need it to be a link and monospaced otherwise. Monospaced links do not get the underline so the reader doesn't know it's a link.

Reply:
That's unfortunate. Anyway, sounds good to me, then.